README copy.md

```diff
@@ -0,0 +1,42 @@
+# Portfolio mobile set up
+This project was done with html and css.
+
+## Built With
+
+- html
+- css
+
+
+## Getting Started
+
+To get a local copy up and run git clone git clone https://github.com/stevenmukama/portfoliomobile.git and follow these simple steps
+
```

Comment:
- [ ] Kindly rename the file to `README.md` instead of `README copy.md` to do it possible to visualize on the repository. 👀 📑